java/code/src/com/redhat/rhn/manager/contentmgmt/test/ContentManagerChannelAlignmentTest.java

```diff
@@ -22,13 +22,15 @@
 import static com.redhat.rhn.domain.role.RoleFactory.ORG_ADMIN;
 import static java.util.Collections.emptyList;
 import static java.util.Collections.singleton;
+import static java.util.Optional.empty;
```

Comment:
There are some unused imports in this one, and maybe in some other test classes as well

Reply:
I'll revisit the tests. Thanks for reminding!